Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Io_image_plugin.cpp

```diff
@@ -1121,46 +1126,65 @@ void convert(Image* image)
   image->image()->wordKind = WK_FLOAT;
 }
 
+
+template <class vtkReader>
+bool
+Io_image_plugin::load_vtk_file(QFileInfo fileinfo, Image* image)
+{
+#ifdef CGAL_USE_VTK
+  vtkNew<vtkReader> reader;
+  reader->SetFileName(fileinfo.filePath().toUtf8());
+  reader->Update();
+  auto vtk_image = reader->GetOutput();
+  vtk_image->Print(std::cerr);
+  *image = CGAL::IO::read_vtk_image_data(vtk_image); // copy the image data
+  return true;
+#else
+  return false;
+#endif
+}
```

Comment:
As far as I can see, there is no need to have that function template as a member function of the class.

```suggestion
template <class vtkReader>
bool
load_vtk_file(QFileInfo fileinfo, Image* image)
{
#ifdef CGAL_USE_VTK
  vtkNew<vtkReader> reader;
  reader->SetFileName(fileinfo.filePath().toUtf8());
  reader->Update();
  auto vtk_image = reader->GetOutput();
  vtk_image->Print(std::cerr);
  *image = CGAL::IO::read_vtk_image_data(vtk_image); // copy the image data
  return true;
#else
  return false;
#endif
}
```

(and remove the declaration in the header).